Marlin/Marlin_main.cpp

```diff
@@ -756,6 +756,7 @@ void loop() {
           // M29 closes the file
           card.closefile();
           SERIAL_PROTOCOLLNPGM(MSG_FILE_SAVED);
+          ok_to_send();
```

Comment:
Doesn't `ok_to_send` get called at the end of `process_next_command`? This will result in two "ok" messages for `M29`.